src/gilded_rose.spec.js

```diff
@@ -124,3 +125,21 @@ describe('updating of conjured items', () => {
     expect(oldCake.quality).toBe(6);
   });
 });
+
+describe('generating ordered lists of items', () => {
+  it('generates an ordered list with one list item', () => {
+    const list = ['Testing'];
+    const testOutput = getHtmlListFromArray(list);
+    expect(testOutput).toBe(`<ol><li>${JSON.stringify('Testing')}</li></ol>`);
+  });
+  it('generates an ordered list with multiple list items', () => {
```

Comment:
These look nice too, and running `npm run test:coverage` shows that the new code has good coverage. The only thing to maybe consider: passing in list items created with the `Item` constructor to reflect what we actually want to see on https://localhost:3000

Reply:
Good point! Didn't think about that. I just added a new test for one of the items.